### DISCUSSION TITLE
Build base on esp32s3 xiao and gps model

### DISCUSSION THREAD

**Discussion by samsamxu** [erstec/APRS-ESP#70] [General]
Hi 
Thanks for your project 
I am wondering build a esp32s3 xiao  with gps no RF model only post to internet 
Is it possible modified? How to do?

**Comment by erstec** [comment]
> Hi Thanks for your project I am wondering build a esp32s3 xiao with gps no RF model only post to internet Is it possible modified? How to do?

In theory it is possible, but need adjustments in targets (creating new target with your mentioned set of features) and in the code

**Reply by samsamxu** [reply]
Thanks for your information, I will try.
